src/Uno.Sdk/targets/Uno.Common.targets

```diff
@@ -115,6 +115,7 @@
 		<_IgnorePlatformFiles Include="$(WindowsProjectFolder)**"
 			Condition=" $(IsWinAppSdk) != 'true' AND '$(WindowsProjectFolder)'!='' " />
 		<Compile Remove="@(_IgnorePlatformFiles)" />
+		<Page Remove="@(_IgnorePlatformFiles)" />
```

Comment:
@jeromelaban Why are these "remove files from other platforms" block repeated a  million times? Also, they're not identical (e.g. this version has a `None Incude=".."` while others `Exclude` instead.

Reply:
I'm not sure to understand the question, where do you see too many repetitions?

Reply:
I'm talking about the ones in this PR (just 4 times, not a million :)). It seems like one one of these should be enough. Maybe the one in Uno.Common.targets.